packages/polaris-viz/src/components/DonutChart/Chart.tsx

```diff
@@ -94,7 +97,7 @@ export function Chart({
 
   const longestLegendWidth = data.reduce((previous, current) => {
     const estimatedLegendWidth = estimateLegendItemWidth(
-      current.name ?? '',
+      `${current.name ?? ''} ${current.data[0].value} `,
```

Comment:
Wouldn't we only want to add the data value if we are showing the value in the legend 🤔 Should we add a check to see if `showLegendValues == true`?